web/packages/build/eslint.config.mjs

```diff
@@ -80,17 +80,23 @@ export default tseslint.config(
           'newlines-between': 'always-and-inside-groups',
         },
       ],
-
+      // typescript-eslint recommends to turn import/no-unresolved off.
+      // https://typescript-eslint.io/troubleshooting/typed-linting/performance/#eslint-plugin-import
+      'import/no-unresolved': 'off',
```

Comment:
I like that you changed numeric values to `'off'`, `'error'`, etc. (it was always super confusing for me). But I think some of the old comments and the order of rules were important, so I brought back the original order and the comments. Let me know if you disagree.